procmgr/processmanager.cpp

```diff
@@ -4961,7 +4962,7 @@ int ProcessManager::addModule(oam::DeviceNetworkList devicenetworklist, std::str
     //clear out the known_host file, sometimes causes a failure on amazon during addModule
     if ( amazon )
     {
-        string cmd = "sudo unlink " + homedir + ".ssh/know_hosts > /dev/null 2>&1";
+        string cmd = "unlink " + homedir + ".ssh/know_hosts > /dev/null 2>&1";
```

Comment:
typo, should be known_hosts not know_hosts.  Also, use 'rm' not 'unlink' for clarity.  Also, this probably doesn't need to be done, and being broken reinforces that suspicion.